composables/helpers/buildPaymentMethodInput.ts

```diff
@@ -0,0 +1,39 @@
+import type { BillingInfo, PaymentActionInput } from "@/server/types/GraphQL"
+import { creditCardPaymentGetters } from "@/lib/getters/creditCardPaymentGetters"
+
+export const buildPaymentMethodInput = (
+  paymentType,
+  currencyCode,
+  paymentWorkflow,
+  checkout,
+  creditCardData,
+  tokenizedData
+): PaymentActionInput => {
+  const ccPaymentServiceCardId = creditCardPaymentGetters.getId(tokenizedData)
+  const ccNumberPart = creditCardPaymentGetters.getCardNumberMask(tokenizedData)
+
+  const paymentActionObjectParams: BillingInfo = {
+    paymentType,
+    card: {
+      paymentServiceCardId: ccPaymentServiceCardId,
+      isUsedRecurring: false,
+      // nameOnCard: creditCardData.cardholderName,
+      isCardInfoSaved: false,
```

Comment:
the value isCardInfoSaved should be based on the input form (what is currently "keepCard")

Reply:
Done